### PR TITLE
Added glance cache cleaning and pruning

### PR DESCRIPTION
**Pull request by angdraug**
This is a rebase of #521.

### CODE REVIEW

File: deployment/puppet/glance/manifests/api.pp [hunk 1]
```diff
@@ -88,43 +108,43 @@
...
+      'DEFAULT/use_stderr':  ensure=> absent;
+      'DEFAULT/use_syslog':  value => true;
+      'DEFAULT/syslog_log_facility': value =>  $syslog_log_facility;
+    }
+    if !defined(File["glance-logging.conf"]) {
       file {"glance-logging.conf":
-         content => template('glance/logging.conf-nosyslog.erb'),
-         path => "/etc/glance/logging.conf",
+        content => template('glance/logging.conf.erb'),
+        path => "/etc/glance/logging.conf",
```
Comment (Zipfer) [inline deployment/puppet/glance/manifests/api.pp:125]: indentation

**Comment by Zipfer** [comment]
If you can you should use single quotes if you don't use interpolation

**Comment by Zipfer** [comment]
+1

**Comment by bogdando** [comment]
+1

**Pull request merged**